examples/helia-101/README.md

```diff
@@ -137,77 +216,18 @@ const libp2p = await createLibp2p({
 })
 ```
 
-#### Helia
-
-Now we have the components we need, we can create our Helia instance and print out some information about the node:
-
-```js
-import { createHelia } from 'helia'
-
-const helia = await createHelia({
-  libp2p,
-  datastore,
-  blockstore
-})
-
-console.info(helia.libp2p.peerId)
-```
-
-#### Putting it all together
-
-Running the code above gets you:
-
-```console
-> node index.js
-PeerId(12D3KooW...)
-```
-
-#### File systems
-
-Blocks are all good and well, but to really get moving we need a filesystem.  Add the `@helia/unixfs` dependency to your project, create an instance and add some data:
-
-```js
-import { unixfs } from '@helia/unixfs`
-
-// create an UnixFS instance by passing the Helia node to the factory function
-const fs = unixfs(helia)
-
-// we will use the `TextEncoder` to turn a string into a Uint8Array
-const encoder = new TextEncoder()
-
-// add the bytes to your Helia node and receive a CID
-const cid = await fs.addBytes(encoder.encode('Hello World 101'))
+### Putting it all together
 
-// let's see what it looks like
-console.log('Added file:', cid)
-```
-
-You can now go to an IPFS Gateway and load the printed hash from a gateway. Go ahead and try it!
+Once your Helia node is configured with a libp2p node, you can go to an IPFS Gateway and load the printed hash. Go ahead and try it!
 
 ```bash
-> node 1.js
-Version: 0.31.2
+> node 301-networking.js
 
 Added file: bafkreife2klsil6kaxqhvmhgldpsvk5yutzm4i5bgjoq6fydefwtihnesa
```

Comment:
Isn't more going to be printed than this since the two nodes are talking to each other?

Reply:
Nothing else is printed by the example, no.  There are more steps and they have more comments around them but there's no other output.

Do you think it should be more verbose?